tools/packaging/common/istio.service

```diff
@@ -8,6 +8,7 @@ ExecStopPost=/usr/local/bin/istio-start.sh clean
 Restart=always
 StartLimitInterval=0
 RestartSec=10
+KillMode=mixed
```

Comment:
from the doc it seems like we would also want TimeoutStopSec set? to control when the SIGKILL is sent after initial SIGTERM?

Reply:
It might be better for clarity, but it's not the cause of the current issue.

E.g., on `Debian 10`, the default value of `TimeoutStopSec` is

```
TimeoutStopUSec=1min 30s
```

Reply:
I agree its not the root cause, but 1m30s is not a great default for this; Kubernetes is 30s

Reply:
Nice suggestion, it is reasonable to keep the same setting with Kubernetes. Code has updated, please have a look, thanks.